frame/ddc-metrics-offchain-worker/src/tests/mod.rs

```diff
@@ -233,7 +227,7 @@ fn should_submit_signed_transaction_on_chain() {
         // Get the transaction from the worker.
         let transactions = pool_state.read().transactions.clone();
         eprintln!("Transactions: {:?}\n", transactions);
-        assert_eq!(transactions.len(), 2);
+        assert_eq!(transactions.len(), 4); // sc_get_current_period_ms + fetch_nodes + (2 x send_metrics_to_sc)
```

Comment:
No, `sc_get_current_period_ms + fetch_nodes` should not make tx, and instead there should be 2 tx for DDN metrics. It would great to check the content (like the report_metrics tx below). You can find the expected numbers from the test_data files.

Reply:
Yes, we already have 2 more tx-s for DDN metrics here. 
I've added checks.